packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -218,11 +257,150 @@ export class TimeSpan {
  * Converts a date string from the Coveo Search API format to the ISO-8601 format.
  * Replace `/` characters in date string with `-`.
  * Replace `@` characters in date string with `T`.
- * @param {string} dateString 
+ * @param {string} dateString
  * @returns {string}
  */
 export function fromSearchApiDate(dateString) {
-  return dateString
-    .replaceAll('/', '-')
-    .replaceAll('@', 'T');
-}
\ No newline at end of file
+  return dateString.replaceAll('/', '-').replaceAll('@', 'T');
```

Comment:
what do you think of grouping these common utils into a class or classes? Given all our utils are in one file I feel it might help organise things.